### PR TITLE
Update host details dashboard to use Series Agg

### PR DESCRIPTION
**Pull request by monicasarbu**
Update the Host details dashboards from the system module of Metricbeat to use Series Agg.

metricbeat host overview kibana

cc-ed @simianhacker

### CODE REVIEW

File: metricbeat/module/system/_meta/kibana/visualization/1aae9140-1b93-11e7-8ada-3df93aab833e.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
...point_size\":1,\"seperate_axis\":0,\"split_mode\":\"everything\",\"stacked\":\"none\",\"value_template\":\"{{value}}/s\",\"terms_field\":\"system.network.name\"},{\"axis_position\":\"right\",\"chart_type\":\"line\",\"color\":\"#68BC00\",\"fill\":0.5,\"formatter\":\"bytes\",\"id\":\"37f70440-1b92-11e7-bec4-a5e9ec5cab8b\",\"label\":\"Total Transfered\",\"line_width\":1,\"metrics\":[{\"field\":\"system.network.out.bytes\",\"id\":\"37f72b50-1b92-11e7-bec4-a5e9ec5cab8b\",\"type\":\"max\"},{\"field\":\"37f72b50-1b92-11e7-bec4-a5e9ec5cab8b\",\"id\":\"37f72b51-1b92-11e7-bec4-a5e9ec5cab8b\",\"type\":\"derivative\",\"unit\":\"1s\"},{\"unit\":\"\",\"id\":\"f9da2dd0-1b92-11e7-a416-41f5ccdba2e6\",\"type\":\"positive_only\",\"field\":\"37f72b51-1b92-11e7-bec4-a5e9ec5cab8b\"},{\"sigma\":\"\",\"field\":\"f9da2dd0-1b92-11e7-a416-41f5ccdba2e6\",\"id\":\"3e63c2f0-1b92-11e7-bec4-a5e9ec5cab8b\",\"type\":\"series_agg\",\"function\":\"overall_sum\"}],\"point_size\":1,\"seperate_axis\":0,\"split_mode\":\"terms\",\"stacked\":\"none\",\"value_template\":\"{{value}}\",\"terms_field\":\"system.network.name\"}],\"show_legend\":1,\"time_field\":\"@timestamp\",\"type\":\"metric\"},\"aggs\":[],\"listeners\":{}}",
```
Comment (simianhacker) [inline metricbeat/module/system/_meta/kibana/visualization/1aae9140-1b93-11e7-8ada-3df93aab833e.json]: The first series in this metric needs to be split by terms on system.network.name and it needs to have a series agg with just a normal sum. The second series looks good (with the terms and series agg overall sum)

File: metricbeat/module/system/_meta/kibana/visualization/522ee670-1b92-11e7-bec4-a5e9ec5cab8b.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
..."point_size\":1,\"seperate_axis\":0,\"split_mode\":\"everything\",\"stacked\":\"none\",\"value_template\":\"{{value}}/s\",\"terms_field\":\"system.network.name\"},{\"axis_position\":\"right\",\"chart_type\":\"line\",\"color\":\"#68BC00\",\"fill\":0.5,\"formatter\":\"bytes\",\"id\":\"37f70440-1b92-11e7-bec4-a5e9ec5cab8b\",\"label\":\"Total Transfered\",\"line_width\":1,\"metrics\":[{\"field\":\"system.network.in.bytes\",\"id\":\"37f72b50-1b92-11e7-bec4-a5e9ec5cab8b\",\"type\":\"max\"},{\"field\":\"37f72b50-1b92-11e7-bec4-a5e9ec5cab8b\",\"id\":\"37f72b51-1b92-11e7-bec4-a5e9ec5cab8b\",\"type\":\"derivative\",\"unit\":\"1s\"},{\"unit\":\"\",\"id\":\"f9da2dd0-1b92-11e7-a416-41f5ccdba2e6\",\"type\":\"positive_only\",\"field\":\"37f72b51-1b92-11e7-bec4-a5e9ec5cab8b\"},{\"sigma\":\"\",\"field\":\"f9da2dd0-1b92-11e7-a416-41f5ccdba2e6\",\"id\":\"3e63c2f0-1b92-11e7-bec4-a5e9ec5cab8b\",\"type\":\"series_agg\",\"function\":\"overall_sum\"}],\"point_size\":1,\"seperate_axis\":0,\"split_mode\":\"terms\",\"stacked\":\"none\",\"value_template\":\"{{value}}\",\"terms_field\":\"system.network.name\"}],\"show_legend\":1,\"time_field\":\"@timestamp\",\"type\":\"metric\"},\"aggs\":[],\"listeners\":{}}",
```
Comment (simianhacker) [inline metricbeat/module/system/_meta/kibana/visualization/522ee670-1b92-11e7-bec4-a5e9ec5cab8b.json]: The first series in this metric needs to be split by terms on system.network.name and it needs to have a series agg with just a normal sum. The second series looks good (with the terms and series agg overall sum)

**Comment by monicasarbu** [comment]
@simianhacker I have addressed your remarks.

**Comment by monicasarbu** [comment]
After the change, the dashboards looks like this:

metricbeat host overview kibana 1

**Comment by ruflin** [comment]
@monicasarbu Awesome dashboards. Thanks.

**Comment by monicasarbu** [comment]
@simianhacker I have done the changes you requested. Can you please have a look?

**Pull request merged**